GitUI/CommandsDialogs/FormRebase.cs

```diff
@@ -314,10 +333,32 @@ private void btnChooseFromRevision_Click(object sender, EventArgs e)
             }
         }
 
-        private void CommitButtonClick(object sender, EventArgs e)
+        private void Commit_Click(object sender, EventArgs e)
         {
             UICommands.StartCommitDialog(this);
             EnableButtons();
         }
+
+        internal TestAccessor GetTestAccessor() => new TestAccessor(this);
+
+        internal readonly struct TestAccessor
+        {
+            private readonly FormRebase _form;
+
+            public TestAccessor(FormRebase form)
+            {
+                _form = form;
+            }
+
+            public CheckBox chkInteractive => _form.chkInteractive;
+            public CheckBox chkPreserveMerges => _form.chkPreserveMerges;
+            public CheckBox chkAutosquash => _form.chkAutosquash;
+            public CheckBox chkStash => _form.chkStash;
+            public CheckBox chkIgnoreDate => _form.chkIgnoreDate;
+            public CheckBox chkCommitterDateIsAuthorDate => _form.chkCommitterDateIsAuthorDate;
+            public void chkIgnoreDate_Click() => _form.chkIgnoreDate_Click(_form.chkIgnoreDate, null);
+            public void chkCommitterDateIsAuthorDate_Click() => _form.chkCommitterDateIsAuthorDate_Click(_form.chkCommitterDateIsAuthorDate, null);
+            public void InteractiveRebaseClick() => _form.InteractiveRebaseClick(_form.chkInteractive, null);
```

Comment:
These are redundant. The handlers are invoked by the virtue of changing the `Checked` state.

Reply:
I thought so at first too, but the `Click` event is not fired by changing the `Checked` property. I even experimented with it in my own project to see, and it seems like only the `CheckChanged` event is fired by changing the `Checked` property programmatically.